.github/actions/build-dotnet/Write-CacheConfiguration.ps1

```diff
@@ -19,8 +18,8 @@ $paths | % { New-Item -ItemType Directory -Force $_ } | Out-Null
 
 # Multiple paths must be separated by "\n", but we can't include newline in the workflow command so we have to misuse
 # the format function like this.
-Set-Output 'paths' ($paths -join '{0}')
+Set-Output('paths' ,($paths -join '{0}'))
 
-Set-Output 'cache-enabled' 'true'
-Set-Output 'key' "${RestoreKeys}-${Hash}"
-Set-Output 'restore-keys' $RestoreKeys
+Set-Output('cache-enabled', 'true')
+Set-Output('key', "${RestoreKeys}-${Hash}")
+Set-Output('restore-keys', $RestoreKeys)
```

Comment:
Why are the brackets and commas needed? `Set-Output` is a script, not a function.